libdnf/hy-util.c

```diff
@@ -157,6 +157,13 @@ hy_is_glob_pattern(const char *pattern)
     return strpbrk(pattern, "*[?") != NULL;
 }
 
+gboolean
+hy_is_file_pattern(const char *pattern)
+{
+    const char *e = pattern +1;
+    return (*pattern == '/') || ((*pattern == '*') && (*e == '/'));
```

Comment:
What about `static inline` function in header file?
One line is enought:
`return pattern[0] == '/' || (pattern[0] == '*' && pattern[1] == '/');`